### PR TITLE
Use P1 for Command-Line Argument Example

### PR DESCRIPTION
**Pull request by jamesjuett**
This PR updates the tutorials to use project 1 instead of project 5 (now project 4) as an example for command-line arguments. This will make it more relevant for students starting EECS 280 and is now possible that P1 uses command-line arguments.

I've targeted this to merge into `p1-tutorial-updates` so that the diff is clearer. If that is merged first, I'll retarget this to `main`.
[Actually - it looks like Github was smart enough to do this automatically.]

The following images need to be updated:

- [x] images/visualstudio149.png
- [x] images/xcode259.png

### CODE REVIEW

File: docs/setup_gdb.md [hunk 1]
```diff
@@ -92,21 +92,22 @@ We recommend enabling the address sanitizer and undefined behavior sanitizer. Th
 
 First, edit your `Makefile` and add the `CXXFLAGS` recommended by the [ASAN Quick Start](setup_asan.html#quick-start).
 
-### Arguments and options
-Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 4:
+### Command-Line Arguments
```
Comment (awdeorio) [inline docs/setup_gdb.md:95]: Do you think that adding "Options" will help avoid confusion when they are used in the future?

```suggestion
### Command-Line Arguments and Options
```

File: docs/setup_gdb.md [hunk 2]
```diff
@@ -92,21 +92,22 @@ We recommend enabling the address sanitizer and undefined behavior sanitizer. Th
...
+Inputs to a program may be provided when it is initailly run via command-line arguments. Here's an example from EECS 280 Project 1:
```
Comment (awdeorio) [inline docs/setup_gdb.md:96]:
```suggestion
Inputs to a program may be provided when it is initially run via command-line arguments or options. Here's an example from EECS 280 Project 1:
```

**Review by awdeorio — APPROVED** [review]
Overall LGTM.  I left a suggestion about mentioning Options, which would apply to each of the modified sections if you decide to accept the suggestion.

**Comment by jamesjuett** [comment]
@awdeorio I had originally left options, as well as adding another example (because we no longer have the naturally occurring `--debug` example from P5), something like this:

> Many programs use *options*, which are arguments starting with `-` or `--`, to customize their behavior. For example, `-v` or `--verbose` tells a program to print more verbose output than default.

However, I felt like it was getting too much into the territory of *teaching about the general conventions* for arguments, options, flags (which are really just options without parameters), etc. (a la https://stackoverflow.com/questions/36495669/difference-between-terms-option-argument-and-parameter).

For the tutorials, I think it is probably best to leave it at *configuring* arguments when you run in the various IDEs or debuggers. (Which covers options and flags as well, since those are just arguments that follow particular conventions.)

But, I don't have a strong preference - LMK what you think about the tradeoff. In particular, I don't have any objection to adding the word "option" to the places you suggested if you think it helps.

**Comment by awdeorio** [comment]
I might be nice to have an indicator for students on later projects that options are configured in the same way as arguments.  From the perspective of C++, options are args, but from the perspective of the CLI, the nomenclature often has them as distinct ([example](https://www.computernetworkingnotes.com/linux-tutorials/linux-commands-options-and-arguments-explained.html)).

**Comment by jamesjuett** [comment]
> I might be nice to have an indicator for students on later projects that options are configured in the same way as arguments. From the perspective of C++, options are args, but from the perspective of the CLI, the nomenclature often has them as distinct ([example](https://www.computernetworkingnotes.com/linux-tutorials/linux-commands-options-and-arguments-explained.html)).

Makes sense. I added your suggestions in dae6b75. Thanks!

**Pull request merged**